dashboard/models.py

```diff
@@ -0,0 +1,166 @@
+
+
+class AcademicTerms(models.Model):
+    term_id = models.AutoField(primary_key=True, verbose_name="Term Id")
+    name = models.CharField(max_length=255, verbose_name="Name")
+    start_date = models.DateField(verbose_name="Start Date")
+    end_date = models.DateField(verbose_name="End Date")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        managed = False
+        db_table = 'academic_terms'
+        verbose_name = "Academic Terms"
+        verbose_name_plural = "Academic Terms"
+
+class Assignment(models.Model):
+    id = models.CharField(primary_key=True, max_length=255, verbose_name="Assignment Id")
+    name = models.CharField(max_length=255, verbose_name="Name")
+    due_date = models.DateTimeField(blank=True, null=True, verbose_name="Due DateTime")
+    local_date = models.DateTimeField(blank=True, null=True, verbose_name="Local DateTime")
+    points_possible = models.CharField(max_length=255, blank=True, null=True, verbose_name="Points Possible")
+    course_id = models.CharField(max_length=255, verbose_name="Course Id")
+    assignment_group_id = models.CharField(max_length=255, verbose_name="Assignment Group Id")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        managed = False
+        db_table = 'assignment'
+
+
+class AssignmentGroups(models.Model):
+    id = models.CharField(primary_key=True, max_length=255, verbose_name="Assignment Group Id")
+    name = models.CharField(max_length=255, verbose_name="Name")
+    weight = models.CharField(max_length=255, blank=True, null=True, verbose_name="Weight")
+    group_points = models.CharField(max_length=255, blank=True, null=True, verbose_name="Group Points")
+    course_id = models.CharField(max_length=255, verbose_name="Course Id")
+    drop_lowest = models.CharField(max_length=255, blank=True, null=True, verbose_name="Drop Lowest")
+    drop_highest = models.CharField(max_length=255, blank=True, null=True, verbose_name="Drop Highest")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        managed = False
+        db_table = 'assignment_groups'
+        verbose_name = "Assignment Groups"
+        verbose_name_plural = "Assignment Groups"
+
+
+class AssignmentWeightConsideration(models.Model):
+    course_id = models.CharField(primary_key=True, max_length=255, verbose_name="Course Id")
+    consider_weight = models.IntegerField(blank=True, null=True, verbose_name="Consider Weight")
+
+    class Meta:
+        managed = False
+        db_table = 'assignment_weight_consideration'
+
+class Course(models.Model):
+    id = models.CharField(primary_key=True, max_length=255, verbose_name="Canvas Course Id")
+    term_id = models.ForeignKey(AcademicTerms, verbose_name="Term Id", on_delete=models.SET_NULL, db_column='term_id')
+    name = models.CharField(max_length=255, verbose_name="Name")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        managed = False
```

Comment:
what does the `managed=False` mean i.e what is the purpose/use?

Reply:
From the above comment, line 6: 
"Remove `managed = False` lines if you wish to allow Django to create, modify, and delete the table"

Reply:
Right, currently these are being created by the init.sql script. If we want the model to create these tables we'll have to modify the script. This could be changed in a future commit.